### PR TITLE
[WIP] Bump go deps to 1.11 for 1.32

### PR DESCRIPTION
**Pull request by skonto**
- similar to #2312 
-  testing

### CODE REVIEW

**Comment by skonto** [comment]
For the lint I havent fixed:

```
$ ~/go/bin/golangci-lint run
test/eventinge2erekt/features/encryption.go:36:45: unused-parameter: parameter 'refs' seems to be unused, consider removing or renaming it as _ (revive)
func VerifyEncryptedTrafficToActivatorToApp(refs []corev1.ObjectReference, since time.Time) *feature.Feature {
                                            ^
test/extensione2erekt/features/encryption.go:18:43: unused-parameter: parameter 'refs' seems to be unused, consider removing or renaming it as _ (revive)
func VerifyEncryptedTrafficForKafkaSource(refs []corev1.ObjectReference, sinkName string, since time.Time) *feature.Feature {
                                          ^
test/extensione2erekt/features/encryption.go:49:43: unused-parameter: parameter 'refs' seems to be unused, consider removing or renaming it as _ (revive)
func VerifyEncryptedTrafficForKafkaBroker(refs []corev1.ObjectReference, since time.Time) *feature.Feature {
                                          ^
test/extensione2erekt/servicemesh/multitenant_test.go:14:2: import "knative.dev/eventing/pkg/apis/duck/v1" imported as "eventingduckv1" but must be "duckv1" according to config (importas)
	eventingduckv1 "knative.dev/eventing/pkg/apis/duck/v1"
	^
```

**Comment by skonto** [comment]
```
12:50:48.003 ERROR:   🚨 Error (code: 1) occurred at ./hack/lib/serverless.bash:406, with command: oc wait --for=condition=Ready knativeeventing.operator.knative.dev knative-eventing -n "${EVENTING_NAMESPACE}" --timeout=900s
12:50:48.004 INFO:    Dumping state...
12:50:48.005 DEBUG:   Environment variables:
```
Seems eventing is not coming up.

**Comment by skonto** [comment]
Basically I noticed in failing tests in the Eventing CR:

```
  Warning  InternalError          8m54s (x17 over 15m)  knativeeventing-controller  failed to apply non rbac manifest: no matches for kind "HorizontalPodAutoscaler" in version "autoscaling/v2beta2"
```

File: vendor/knative.dev/operator/pkg/reconciler/knativeeventing/knativeeventing.go [hunk 1]
```diff
@@ -143,14 +144,27 @@ func (r *Reconciler) transform(ctx context.Context, manifest *mf.Manifest, comp
 		kec.ReplicasEnvVarsTransform(manifest.Client),
 	}
 	extra = append(extra, r.extension.Transformers(instance)...)
-	return common.Transform(ctx, manifest, instance, extra...)
+	return common.Transform(ctx, manifest, instance)
```
Comment (skonto) [inline vendor/knative.dev/operator/pkg/reconciler/knativeeventing/knativeeventing.go:147]: @pierDipi I think here we missed `extra` correct?

Comment (pierDipi) [reply vendor/knative.dev/operator/pkg/reconciler/knativeeventing/knativeeventing.go:147]: I'm not sure why it was removed

Comment (pierDipi) [reply vendor/knative.dev/operator/pkg/reconciler/knativeeventing/knativeeventing.go:147]: I'm adding it back https://github.com/knative/operator/pull/1587

Comment (pierDipi) [reply vendor/knative.dev/operator/pkg/reconciler/knativeeventing/knativeeventing.go:147]: Thanks for the good catch!

**Comment by skonto** [comment]
@pierDipi any idea why the Eventing tests are failing here?

**Comment by pierDipi** [comment]
I fixed them in my PR https://github.com/openshift-knative/serverless-operator/pull/2312 that's why the diff here https://github.com/openshift-knative/serverless-operator/pull/2312#discussion_r1351853256

**Pull request closed without merging**